### DISCUSSION TITLE
选集区域优化建议

### DISCUSSION THREAD

**Discussion by pinaZC** [the1812/Bilibili-Evolved#4657] [功能建议 / Ideas]
目前这个功能的体验就是：不展开 标题显示不全，也显示不了几条视频，想要的信息太少；（原版体验）
只展开标题 标题全了，但是可以显示的视频量更少了，想找感兴趣的内容很费力；
展开 又有很多懒狗up搞得100+合集，还各种不按时间排序，这个区域也没有图片，纯文字，还占很大的界面，信息量就又太多了，同样需要更费力才能找到想看的内容；
找到想看的内容后 alt+左键手动收起，列表会直接返回到置顶，还是要去翻自己看到哪了。

预想比较好的效果：
只展开到固定或者与播放器高度一致的选集区域，同时展开标题 这样原本的在播放视频定位功能应该也不会失效，也能控制这个区域的版面和信息量
（不知道这样是否好实现，虽然合集就是破站的破功能，但既然优化了还是希望能做得体验更好一些，谢谢大佬）
![image](https://github.com/the1812/Bilibili-Evolved/assets/72682556/cf09615e-939b-4166-87be-b50086dac078)

**Comment by Geeyun-JY3** [comment]
还真别说，我几年前提的针对这个功能的前身的优化建议时提过这个对齐播放器，尽管对象和思路不太一样：#692
